src/patternfly/components/DataList/examples/data-list-check-action-example.hbs

```diff
@@ -0,0 +1,24 @@
+{{#> data-list data-list--modifier='pf-u-box-shadow-md' data-list--attribute='aria-label="Checkbox and action data list example"'}}
```

Comment:
Is the box shadow just for the workspace example, or should the list itself have a box shadow? If it's just for the workspace, I wonder if we should use workspace styling to draw some sort of border or box shadow so someone won't copy/paste the utility class into their app. If the list itself should have a box shadow, I think that CSS should be added to the component.

Reply:
This brings up a convo @andresgalante and I had about including box shadows in the component vs applying utility classes. All examples have box shadows, but there may be a case where they aren't needed/wanted? On the other hand, adding them to the component would drive consistency.. I prefer adding the shadow to the component.

Reply:
Gotcha. If the designs have them, and we're introducing a utility class for an edge case where they aren't needed, that says to me that 80% of the time the component should have them, so they should just come with the component without any non-component classes. And if someone wants to remove them, they can overwrite a variable. That's assuming that the default presentation should have shadows tho. Maybe that's something we could double check with @kybaker  about? 

If it's like 50/50 as to whether the component should have a box shadow, depending on where it's placed or whatever, I would argue we should remove the utility classes from the component example, and use utility classes in a demo illustrating a case where you might want to add a box shadow.